### DISCUSSION TITLE
数据标签Label声明到视图层级，可以排除掉个别子项吗

### DISCUSSION THREAD

**Discussion by xyr550** [antvis/G2#5600] [Q&A]
**场景如下**

图表展示多组柱子，柱子均有标签，多组柱子的标签需要隐藏重叠标签（加在了整个视图层， 否则两组柱子上的标签会有重叠）

但是用lineY画了两条标注线，标注线的label，也因为视图上的配置标签被隐藏了，代码结构如下

```js
option = {
  type: 'view',
  labelTransform: [{ type: 'overlapHide '}],
  children:[
    {
      type:' interval'
    },
    {
      type:' interval'
    }，
    {
      type: 'lineY'，
      labels：[{
         text: 'kmksdncs',
         transform: []
      }]
    }，
   {
      type: 'lineY'
    }
  ]
}
```

**如何实现，标签的设置 labelTransform 对某些子视图不生效，lineY画的辅助线的label全都不隐藏，就叠着绘出来**

![1695711026368](https://github.com/antvis/G2/assets/73283715/1c2a16f4-7115-4a4d-bf5d-52cdadbea3aa)

想要效果
![image](https://github.com/antvis/G2/assets/73283715/b2aa13ba-7d78-4baa-967f-75e50fd2d70e)

**Comment by pearmini** [comment]
目前还不支持这个能力，也许可以增加一个配置解决这个问题?

```js
const option = {
  type: 'view',
  labelTransform: [{ type: 'overlapHide ' }],
  children: [
    {
      type: 'lineY',
      labels: [
        {
          text: 'kmksdncs',
          skipTransform: true, // 这样
          transform: [],
        },
      ],
    },
  ],
};
```
